bootstrap.sh

```
@@ -242,6 +241,195 @@ function getUbuntuVersion {
+            docker-compose-plugin
+    else
+        logStep "Docker already installed."
+    fi
+}
+
+function stepInstallDockerCompose() {
+    # https://github.com/docker/compose
+    local DOCKER_CLI_PLUGINS_DIR="/usr/local/lib/docker/cli-plugins"
+    local DOCKER_COMPOSE_BIN="${DOCKER_CLI_PLUGINS_DIR}/docker-compose"
+    local DOCKER_COMPOSE_REPO="docker/compose"
+    local DOCKER_COMPOSE_ASSET="docker-compose-linux-$(uname -m)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)